### PR TITLE
Cleanup gitignore

### PR DESCRIPTION
**Pull request by gsnedders**
This change is [Reviewable](https://reviewable.io/reviews/w3c/web-platform-tests/9981)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,20 +1,29 @@
...
+# WPT repo stuff
+/MANIFEST.json
+
+testharness_runner.html
+!/tools/wptrunner/wptrunner/testharness_runner.html
+
+_certs
+config.json
+
+# Various OS/editor specific files
```
Comment (gsnedders) [inline .gitignore]: Do we want to add a comment about some vague policy about adding stuff here? I mean, I'm assuming we don't just want to add everything from https://github.com/github/gitignore/tree/master/Global into here.

File: .gitignore [hunk 2]
```diff
@@ -1,20 +1,29 @@
...
 *~
-MANIFEST.json
-!content-security-policy/support/manifest.json
-!payment-handler/manifest.json
 \#*
-_certs
-.virtualenv
-config.json
-node_modules
 scratch
```
Comment (gsnedders) [inline .gitignore]: What even is this? Should this be in the previous section?

File: .gitignore [hunk 3]
```diff
@@ -1,20 +1,29 @@
...
-testharness_runner.html
 webdriver/.idea
 .vscode/
 .DS_Store
 *.rej
-_venv
 webdriver/.cache
```
Comment (gsnedders) [inline .gitignore]: What even is this? Should this be in the previous section?

**Comment by w3c-bots** [comment]
# Build PASSED

Started: 2018-03-12 17:26:52
Finished: 2018-03-12 17:33:13





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/25806)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/352449902)

File: .gitignore (hunk 3 above)
Comment (jgraham) [reply .gitignore]: pytest junk

File: .gitignore [hunk 4]
```diff
@@ -1,20 +1,30 @@
...
-!content-security-policy/support/manifest.json
-!payment-handler/manifest.json
 \#*
-_certs
-.virtualenv
-config.json
-node_modules
 scratch
-testharness_runner.html
 webdriver/.idea
```
Comment (jgraham) [inline .gitignore]: I imagine this should be matched anywhere?

File: .gitignore (hunk 2 above)
Comment (jgraham) [reply .gitignore]: No idea

File: .gitignore (hunk 1 above)
Comment (jgraham) [reply .gitignore]: "emacs is the one true editor" seems like a good policy to me :p

I expect it's not worth thinking too hard about this and we should just let the section grow in an ad-hoc way.

**Review by jgraham — APPROVED** [review]

File: .gitignore (hunk 3 above)
Comment (gsnedders) [reply .gitignore]: oh is this where `.pytest_cache` comes from? that, but renamed?

Comment (jgraham) [reply .gitignore]: Yeah it seems that the name changed at some point, so maybe this rule should be updated.

File: .gitignore (hunk 2 above)
Comment (gsnedders) [reply .gitignore]: Seems to be from 8267f079550cbe16e488fa85a04d7d7d309f6529, added along with node. But Google suggests it isn't node, so who knows.

**Comment by gsnedders** [comment]
@jgraham plz merge if this still looks good to you once CI passes

**Review by jgraham — APPROVED** [review]

**Pull request merged**